bin/gen-crons.py

```diff
@@ -23,6 +23,8 @@ def main():
                             'Only define for cron.d style crontabs.'))
     parser.add_option('-p', '--python', default='python2.6',
                       help='Python interpreter to use.')
+    parser.add_option('-n', '--nologdir', action='store_true',
```

Comment:
We could change https://github.com/mozilla/bedrock/blob/master/bin/update/deploy_base.py#L213-L219 to something like the following:

```python
def generate_cron_file(ctx, tmpl_name):
    with ctx.lcd(settings.WWW_DIR):
        ctx.local("CRON_LOG_DIR={cron_log_dir} {python} bin/gen-crons.py "
                  "-p {python} -s {src_dir} -w {www_dir} -t {template}".format(
                    cron_log_dir=getattr(settings, 'CRON_LOG_DIR',
                                         '/var/log/bedrock'),
                    python=PYTHON,
                    src_dir=settings.SRC_DIR,
                    www_dir=settings.WWW_DIR,
                    template=tmpl_name))
```